python311Packages.sentry-sdk: 1.25.0 -> 1.25.1

fabaff opened the pull request:
Diff: https://github.com/getsentry/sentry-python/compare/refs/tags/1.25.0...1.25.1

Changelog: https://github.com/getsentry/sentry-python/blob/1.25.1/CHANGELOG.md

###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff commented:
Result of `nixpkgs-review pr 237299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
  </ul>
</details>
<details>
  <summary>63 packages built:</summary>
  <ul>
    <li>alerta-server</li>
    <li>alerta-server.dist</li>
    <li>cura</li>
    <li>gns3-gui</li>
    <li>gns3-gui.dist</li>
    <li>gns3-server</li>
    <li>gns3-server.dist</li>
    <li>home-assistant-component-tests.sentry</li>
    <li>ledfx</li>
    <li>ledfx.dist</li>
    <li>moodle-dl</li>
    <li>moodle-dl.dist</li>
    <li>mozphab</li>
    <li>mozphab.dist</li>
    <li>netbox</li>
    <li>notus-scanner (python310Packages.notus-scanner)</li>
    <li>notus-scanner.dist (python310Packages.notus-scanner.dist)</li>
    <li>octoprint</li>
    <li>octoprint.dist</li>
    <li>openai-full</li>
    <li>openai-full.dist</li>
    <li>ospd-openvas</li>
    <li>ospd-openvas.dist</li>
    <li>peering-manager</li>
    <li>python310Packages.django-rq</li>
    <li>python310Packages.django-rq.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.k-diffusion</li>
    <li>python310Packages.k-diffusion.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.sentry-sdk</li>
    <li>python310Packages.sentry-sdk.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-loggers</li>
    <li>python310Packages.spacy-loggers.dist</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.wagtail-localize</li>
    <li>python310Packages.wagtail-localize.dist</li>
    <li>python310Packages.wandb</li>
    <li>python310Packages.wandb.dist</li>
    <li>python311Packages.django-rq</li>
    <li>python311Packages.django-rq.dist</li>
    <li>python311Packages.notus-scanner</li>
    <li>python311Packages.notus-scanner.dist</li>
    <li>python311Packages.sentry-sdk</li>
    <li>python311Packages.sentry-sdk.dist</li>
    <li>python311Packages.wagtail-localize</li>
    <li>python311Packages.wagtail-localize.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>tribler</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>

samuela commented on pkgs/development/python-modules/wandb/default.nix:
```diff
@@ -256,9 +256,14 @@ buildPythonPackage rec {
-  # https://github.com/giampaolo/psutil/issues/1219
-  disabledTests = lib.optionals stdenv.isDarwin [
+  disabledTests = [
+    # Time-sensitive, fails on Hydra on a regular base
```
```suggestion
    # Time-sensitive, fails on Hydra on a regular basis
```

fabaff replied:
Thanks, fixed.

samuela reviewed (commented):
besides minor nit LGTM, I think we can merge once all checks go green

fabaff commented:
Result of `nixpkgs-review pr 237299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
  </ul>
</details>
<details>
  <summary>63 packages built:</summary>
  <ul>
    <li>alerta-server</li>
    <li>alerta-server.dist</li>
    <li>cura</li>
    <li>gns3-gui</li>
    <li>gns3-gui.dist</li>
    <li>gns3-server</li>
    <li>gns3-server.dist</li>
    <li>home-assistant-component-tests.sentry</li>
    <li>ledfx</li>
    <li>ledfx.dist</li>
    <li>moodle-dl</li>
    <li>moodle-dl.dist</li>
    <li>mozphab</li>
    <li>mozphab.dist</li>
    <li>netbox</li>
    <li>notus-scanner (python310Packages.notus-scanner)</li>
    <li>notus-scanner.dist (python310Packages.notus-scanner.dist)</li>
    <li>octoprint</li>
    <li>octoprint.dist</li>
    <li>openai-full</li>
    <li>openai-full.dist</li>
    <li>ospd-openvas</li>
    <li>ospd-openvas.dist</li>
    <li>peering-manager</li>
    <li>python310Packages.django-rq</li>
    <li>python310Packages.django-rq.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.k-diffusion</li>
    <li>python310Packages.k-diffusion.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.sentry-sdk</li>
    <li>python310Packages.sentry-sdk.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-loggers</li>
    <li>python310Packages.spacy-loggers.dist</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.wagtail-localize</li>
    <li>python310Packages.wagtail-localize.dist</li>
    <li>python310Packages.wandb</li>
    <li>python310Packages.wandb.dist</li>
    <li>python311Packages.django-rq</li>
    <li>python311Packages.django-rq.dist</li>
    <li>python311Packages.notus-scanner</li>
    <li>python311Packages.notus-scanner.dist</li>
    <li>python311Packages.sentry-sdk</li>
    <li>python311Packages.sentry-sdk.dist</li>
    <li>python311Packages.wagtail-localize</li>
    <li>python311Packages.wagtail-localize.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>tribler</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>

vcunat commented:
Nit about commit naming: `python311Packages.wandb` doesn't really evaluate
```
error: tensorboard-2.11.0 not supported for interpreter python3.11
```

I at least took the update to `staging-next` branch now, as it wouldn't even build without it due to updated python311.

fabaff committed:
python311Packages.sentry-sdk: 1.25.1 -> 1.26.0

Diff: https://github.com/getsentry/sentry-python/compare/refs/tags/1.25.1...1.26.0

Changelog: https://github.com/getsentry/sentry-python/blob/1.26.0/CHANGELOG.md

fabaff commented:
Result of `nixpkgs-review pr 237299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>netbox_3_3</li>
    <li>peering-manager</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
  </ul>
</details>
<details>
  <summary>64 packages built:</summary>
  <ul>
    <li>alerta-server</li>
    <li>alerta-server.dist</li>
    <li>cura</li>
    <li>gns3-gui</li>
    <li>gns3-gui.dist</li>
    <li>gns3-server</li>
    <li>gns3-server.dist</li>
    <li>home-assistant-component-tests.sentry</li>
    <li>ledfx</li>
    <li>ledfx.dist</li>
    <li>moodle-dl</li>
    <li>moodle-dl.dist</li>
    <li>mozphab</li>
    <li>mozphab.dist</li>
    <li>netbox</li>
    <li>notus-scanner (python310Packages.notus-scanner)</li>
    <li>notus-scanner.dist (python310Packages.notus-scanner.dist)</li>
    <li>octoprint</li>
    <li>octoprint.dist</li>
    <li>openai-full</li>
    <li>openai-full.dist</li>
    <li>ospd-openvas</li>
    <li>ospd-openvas.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python310Packages.django-rq</li>
    <li>python310Packages.django-rq.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.k-diffusion</li>
    <li>python310Packages.k-diffusion.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.sentry-sdk</li>
    <li>python310Packages.sentry-sdk.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-loggers</li>
    <li>python310Packages.spacy-loggers.dist</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.wagtail-localize</li>
    <li>python310Packages.wagtail-localize.dist</li>
    <li>python310Packages.wandb</li>
    <li>python310Packages.wandb.dist</li>
    <li>python311Packages.django-rq</li>
    <li>python311Packages.django-rq.dist</li>
    <li>python311Packages.notus-scanner</li>
    <li>python311Packages.notus-scanner.dist</li>
    <li>python311Packages.sentry-sdk</li>
    <li>python311Packages.sentry-sdk.dist</li>
    <li>python311Packages.wagtail-localize</li>
    <li>python311Packages.wagtail-localize.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>tribler</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
